lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -1870,6 +1872,28 @@ namespace Js
                 entypointInfo->SetModuleAddress((uintptr_t)moduleMemoryPtr);
                 funcObj->SetEnvironment(frameDisplay);
                 localModuleFunctions[i] = funcObj;
+                if (functionArray[i]->wasmInfo->Exported())
+                {
+                    PropertyRecord const * propertyRecord = nullptr;
+                    LPCUTF8 name = wasmScript->module->exports->Lookup(i, nullptr);
+                    if (!name)
+                    {
+                        // TODO: can this happen?
+                        Assert(UNREACHED);
```

Comment:
Ah, looks like analysis isn't happy I'm asserting against nullptr but not actually doing anything about it. Fair enough. Guess I'll have to get to that todo sooner than later.